build/stf-run-ci/tasks/setup_stf_local_build.yml

```diff
@@ -19,7 +19,7 @@
         OPERATOR_TAG={{ sgo_image_path | parse_tag | quote }} \
         ./generate_bundle.sh
     register: generate_bundle_sgo
-  always:
+  rescue:
     # "|| true" is needed until https://github.com/infrawatch/smart-gateway-operator/pull/143 is merged
```

Comment:
[#143 is ](https://github.com/infrawatch/smart-gateway-operator/pull/143)  is merged now - can this rescue block go away? What's it for, anyways, doesn't ansible.builtin.shell succeed even if the command itself exits non-zero?

Reply:
Nope, a rc != 0 means a failed task
```
---
- name: Test non-zero rc
  hosts: localhost
  tasks:
    - name: Use shell
      ansible.builtin.shell:
        cmd: /bin/false
      register: cmd_out
```

Reply:
Don't trust, verify! Thanks :)